h5grove/flask_utils.py

```diff
@@ -21,10 +22,17 @@
 ]
 
 
-def make_encoded_response(content, format_arg: Optional[str] = "json") -> Response:
+class H5GroveException(HTTPException):
+    def __init__(self, content, status) -> None:
+        super().__init__(response=make_encoded_response(content, status=status))
```

Comment:
...but we would have to make it rethrow a `werkzeug.HTTPException` to work in flask